src/modules/hunting/kubelet.py

```diff
@@ -285,8 +288,9 @@ def test_handlers(self):
                     self.publish_event(ExposedContainerLogsHandler())
                 if debug_handlers.test_exec_container():
                     self.publish_event(ExposedExecHandler())      
-                if debug_handlers.test_run_container():
-                    self.publish_event(ExposedRunHandler())
+                r = debug_handlers.test_run_container()
+                if r:
+                    self.publish_event(ExposedRunHandler(r))
```

Comment:
We're publishing a vulnerability anyway, so why not add the evidence? 

But what I'm more worried about now you're highlighting that active hunter - is this doing a bunch of state-changing things under a regular Hunter rather than an ActiveHunter? I don't think I'm making it any worse with this PR if so, but if that's true, we need to fix it.